addons/pos_self_order/static/tests/tours/tour_utils.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module */
+
+// HELPERS ////////////////////////////////
+// Each of the export functions below returns an array of steps
+// (even those that return a single step, for consistency)
+
+/**
+ * START: product list screen
+ * END: product list screen
+ * @param {int[]} product_ids
```

Comment:
```suggestion
 * @param {number[]} product_ids
```

no `int` type in js.